### ISSUE TITLE
Feature: Show current age

### ISSUE THREAD

**Issue by User1l0** [m-i-n-a-r/birday#364]
It only shows the age for next year. It would be nice to also see the current age of a person

**Comment by herrbrixter** [comment]
I agree. Also please clarify what age is displayed. Now it is confusing if it is the current age or age on next birthday. I suggest a toggle:

1. Show current age only. "40 today"
2. Show next age only. "Will be 41”
3. Show both ages. "40 going to 41"

**Comment by m-i-n-a-r** [comment]
Sorry for the tremendous amount of delay. Yep, it makes sense but it would require a lot of new translations, I'll see what I can do to make it less confusing.
